.github/workflows/build_x86_64.yml

```diff
@@ -0,0 +1,94 @@
+#=================================================
+# Description: Test build OpenWrt using GitHub Actions
+# Lisence: MIT
+# Author: Xiaobo Tian peterwillcn@gmail.com
+#=================================================
+
+name: Build_x86_64
+
+on: [push, pull_request]
+#  push:
+#    branches:
+#      - master
+
+  #schedule:
+  #  - cron: 0 8 * * 5
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@master
+
+    - name: Initialization environment
+      env:
+        DEBIAN_FRONTEND: noninteractive
+      run: |
+        docker rmi `docker images -q`
+        echo "Deleting files, please wait ..."
+        sudo rm -rf \
+          /usr/share/dotnet \
+          /etc/mysql \
+          /etc/php
+        sudo -E apt-get -y purge \
+          azure-cli \
+          ghc* \
+          zulu* \
+          hhvm \
+          llvm* \
+          firefox \
+          google* \
+          dotnet* \
+          powershell \
+          openjdk* \
+          mysql* \
+          php*
+        sudo -E apt-get update
+        sudo -E apt-get -y install build-essential asciidoc binutils bzip2 gawk gettext git libncurses5-dev patch unzip zlib1g-dev lib32gcc1 libc6-dev-i386 subversion flex node-uglify gcc-multilib p7zip p7zip-full msmtp libssl-dev texinfo libglib2.0-dev xmlto qemu-utils upx-ucl libelf-dev autoconf automake libtool autopoint device-tree-compiler libuv-dev
+        sudo -E apt-get -y autoremove --purge
+        sudo -E apt-get clean
```

Comment:
Is there a way to use a container ? so we avoid reinstalling the base package each time.

Reply:
Huh. Why purge all those packages?

Reply:
> Is there a way to use a container ? so we avoid reinstalling the base package each time.

I want to change to buildbot's docker images, but I'm having trouble with it at the moment

Reply:
> Huh. Why purge all those packages?

Saves space for compiling the image.